src/sanitizer.js

```diff
@@ -198,7 +198,8 @@ const INVALID_INLINE_STYLE_REGEX =
  * @return {string}
  */
 export function sanitizeHtml(html) {
-  const tagPolicy = htmlSanitizer.makeTagPolicy();
+  const tagPolicy = htmlSanitizer.makeTagPolicy(parsed =>
+    parsed.scheme_ === 'https' ? parsed : null);
```

Comment:
I had to change this a bit from using `isSecureUrl` since the callback gets a `URI` object, not an url